### PR TITLE
Add test node and update testing instructions

### PR DESCRIPTION
**Pull request by livanov93**

### CODE REVIEW

File: ur_ros2_control_demos/CMakeLists.txt [hunk 1]
```diff
@@ -2,9 +2,21 @@ cmake_minimum_required(VERSION 3.5)
 project(ur_ros2_control_demos)
 
 find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+#find_package(std_srvs REQUIRED)
```
Comment (destogl) [inline ur_ros2_control_demos/CMakeLists.txt:6]: If not needed delete...
```suggestion
```

File: ur_ros2_control_demos/src/test_driver.cpp [hunk 2]
```diff
@@ -0,0 +1,89 @@
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+#include "std_msgs/msg/float64_multi_array.hpp"
+#include <sensor_msgs/msg/joint_state.hpp>
+#include <std_msgs/msg/multi_array_dimension.hpp>
```
Comment (destogl) [inline ur_ros2_control_demos/src/test_driver.cpp:10]: Keep the style and alphabetic order...

```suggestion
#include "sensor_msgs/msg/joint_state.hpp"
#include "std_msgs/msg/float64_multi_array.hpp"
#include "std_msgs/msg/multi_array_dimension.hpp"
#include "std_msgs/msg/string.hpp"
```

File: ur_ros2_control_demos/src/test_driver.cpp [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  sensor_msgs::msg::JointState msg_internal_;
+};
+
+int main(int argc, char* argv[])
+{
+  rclcpp::init(argc, argv);
+  rclcpp::spin(std::make_shared<MinimalDriverTest>());
+  rclcpp::shutdown();
+  return 0;
+}
```
Comment (destogl) [inline ur_ros2_control_demos/src/test_driver.cpp:89]: Add an empty line to the end of the file...
```suggestion
}

```

**Review by destogl — APPROVED** [review]
Only a few nitpicks. otherwise looks good.

*NOTE*: I didn't test the build procedure. Should I?

**Comment by livanov93** [comment]
> Only a few nitpicks. otherwise looks good.
> 
> 
> 
> *NOTE*: I didn't test the build procedure. Should I?

CI will take care of that

**Pull request merged**